### ISSUE TITLE
Filters?

### ISSUE THREAD

**Issue by Stan92** [EddyVerbruggen/nativescript-bluetooth#180]
Hi,

Nativescript-bluetooth version : 3.0.0-beta.10

Using a filter to the startScanning method, I can't find my devices.
Do I miss something?

```
       const filters=[{serviceUUID:'5001'}];
        const seconds=3;
        bluetooth.startScanning(
            {
             filters,
             seconds,
              onDiscovered:(peripheral:Peripheral)=>{
                   console.log(peripheral);
              },
              skipPermissionCheck:false
        }).then(
            (res)=>{
                console.log("ble scan terminé");
            },
            (err)=>{
                console.log("err",err,true);
            });

```

**Comment by manojdcoder** [comment]
What kind of device you are trying to connect? Where did you find the service UUID from? `5001` is something not listed in the [spec](https://www.bluetooth.com/specifications/gatt/services/).

**Comment by Stan92** [comment]
It’s a esp32 part with a customized code

**Comment by manojdcoder** [comment]
Do you find the specific device on list if you do not apply filters?

**Comment by Stan92** [comment]
Yes I can see it, connect, read the services and characteristics 
Everything works except the filters
Btw thanks for the package

**Comment by manojdcoder** [comment]
Did you test on both iOS & Android? Is it platform specific?

**Comment by Stan92** [comment]
I only tested on Android.

**Comment by manojdcoder** [comment]
Are you able to perform a read / write with the service id?

**Comment by Stan92** [comment]
Yes for the read (I don't have a write)
Everything perfectly works except the filters :) 
I have also noticed, the Peripheral doesn't have the state property (within the interface).
But if I log the object, I get these following properties
 type,UUID,name,RSSI,localName,state,manufacterId,advertismentData

**Comment by Stan92** [comment]
Hi,
It seems it doesn't work too with iOS (probably I missed something).
I also tried with a "standard" service UUID (coming from my MacBook Pro).
